tests/Query/TermsTest.php

```diff
@@ -96,4 +97,72 @@ public function testFilteredSearchWithLookup(): void
 
         $this->assertEquals(2, $resultSet->count());
     }
+
+    /**
+     * @group functional
+     */
+    public function testVariousDataTypesViaConstructor(): void
```

Comment:
Can you add a mixed test too? `Terms('some_numeric_field', ['9876', 1234])`

Reply:
Done! => `testVariousMixedDataTypesViaConstructor`